SYCL/DeviceCodeSplit/aot-gpu.cpp

```diff
@@ -2,6 +2,9 @@
 // UNSUPPORTED: cuda
 // CUDA does neither support device code splitting nor SPIR.
 //
+// The test is failing with GPU RT 30.0.100.9667
+// XFAIL: windows
```

Comment:
Given the list of fails, should something like this be enforced?
```suggestion
// XFAIL: windows && opencl
```

Reply:
Sorry, not for this test, but rather for `multiple-devices.cpp`

Reply:
It is redundant as soon as we have REQUIRES: cpu+gpu+acc which is true only for opencl.
The test is effectively unsupported on CUDA. ROCm, Level Zero